lib/Ast_to_Mir.ml

```diff
@@ -92,7 +117,7 @@ let rec trans_stmt {Ast.stmt_typed; stmt_typed_loc; _} =
         let rhs =
           match assign_op with
           | Ast.Assign | Ast.ArrowAssign -> rhs
-          | Ast.OperatorAssign op -> BinOp (assignee, op, rhs)
+          | Ast.OperatorAssign op -> BinOp (assignee, trans_op op, rhs)
```

Comment:
As discussed yesterday, this is OK for now. But let's keep it in mind as a potential source of optimization.